.github/scripts/pr-builder.sh

```diff
@@ -59,7 +59,7 @@ if [ "$REPO" = "product-is" ]; then
 
   cat pom.xml
   export JAVA_HOME=$JAVA_11_HOME
-  mvn clean install --batch-mode | tee mvn-build.log
+  mvn clean install -Dmaven.test.skip=true -Djdk.util.zip.disableZip64ExtraFieldValidation=true --batch-mode | tee mvn-build.log
```

Comment:
Do we need to skip tests here ?

Reply:
that's been added by mistake, this is fixed now